docs/examples/google-core-iot/README.md

```diff
@@ -48,7 +49,10 @@ $ gcloud iot devices create "$DEVICE_ID" \
     --public-key path=device.crt,type=es256-x509-pem
 ```
 
-**Note:** Google's Device ID [must start with a letter ([a-zA-Z]))](https://cloud.google.com/iot/docs/requirements#permitted_characters_and_size_requirements), so you might need to prefix your devices, as the WoTT Device ID can start with a latter.
+**Note:** Google's Device ID [must start with a letter ([a-zA-Z]))](https://cloud.google.com/iot/docs/requirements#permitted_characters_and_size_requirements).
```

Comment:
Move this up to below the `sed` command.

Reply:
Have reorganised as per request